docs/products/kafka/howto/enable-governance.md

```diff
@@ -3,34 +3,36 @@ title: Enable governance for Aiven for Apache Kafka®
 sidebar_label: Enable governance
 limited: true
 ---
-import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons"
+
 import RelatedPages from "@site/src/components/non-swizzled/RelatedPages";
+import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons"
 
-Enable governance in Aiven for Apache Kafka® to establish a secure and compliant framework to manage your Aiven for Apache Kafka clusters efficiently.
+Enable governance in Aiven for Apache Kafka® to create a secure and compliant framework to manage your Aiven for Apache Kafka services efficiently.
+
+By default, governance applies at the organization level. Admins can manually select
```

Comment:
nit: I'd put the full role name organization admin just because the roles/permissions are more complex now and until very recently there's only been an "admin" role at the project level. The term decided on for all of these even when plural was "admin", too. :)